backpack/extensions/firstorder/sum_grad_squared/__init__.py

```diff
@@ -1,20 +1,33 @@
-from torch.nn import Conv2d, Linear
+from torch.nn import (
+    Conv1d,
+    Conv2d,
+    Conv3d,
+    ConvTranspose1d,
+    ConvTranspose2d,
+    ConvTranspose3d,
+    Linear,
+)
 
 from backpack.extensions.backprop_extension import BackpropExtension
 
-from . import conv2d, linear
+from . import (
+    conv1d,
+    conv2d,
+    conv3d,
+    convtranspose1d,
+    convtranspose2d,
+    convtranspose3d,
+    linear,
+)
 
 
 class SumGradSquared(BackpropExtension):
     """The sum of individual-gradients-squared, or second moment of the gradient.
-
     Stores the output in ``sum_grad_squared``. Same dimension as the gradient.
-
     Note: beware of scaling issue
         The second moment depends on the scaling of the overall function.
         Let ``fᵢ`` be the loss of the ``i`` th sample, with gradient ``gᵢ``.
         ``SumGradSquared`` will return the sum of the squared
-
```

Comment:
Was there a reason why these lines had to go? I think they are necessary to prettify the [docs](https://docs.backpack.pt/en/master/extensions.html#backpack.extensions.SumGradSquared)

Reply:
I think they were removed during `black`. I dont recall removing lines

Reply:
fixed